testcases/SbePassThrough.py

```diff
@@ -80,7 +80,7 @@ def is_sel_sent_to_bmc(self):
 
     def runTest(self):
         self.cv_SYSTEM.goto_state(OpSystemState.OS)
-        self.c = self.cv_SYSTEM.sys_get_ipmi_console()
+        self.c = self.cv_SYSTEM.cv_HOST.get_ssh_connection()
```

Comment:
Here also keep console as interface to run commands

Reply:
Fixed